doc/source/.res/replace.txt

```diff
@@ -1,7 +1,7 @@
 .. |d11n| replace:: documentation
 
-.. |tip.run-all.root|  replace:: Run the following commands as root or by using the :program:`sudo` command
-.. |tip.run-this.root| replace:: Run this command as root or by using the :program:`sudo` command
+.. |tip.run-all.root|  replace:: Run the following commands as root or by using the :program:`sudo` command.
```

Comment:
Can we not use the replace.txt? Put the statements in the doc.

Reply:
fixed